packages/editor/src/components/post-author/post-author-combobox.js

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
Note: this file is moved unchanged from packages/editor/src/components/post-author/iundex.js